compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lowering_strategy_objectfifo_npu4.mlir

```diff
@@ -40,3 +41,54 @@ module {
     return
   }
 }
+
+// -----
+
+// Tests a matmul shape for pack-peel-4-level-tiling in which the tile size generated
+// should ideally be equal to the M,N,K dimensions of the matmul - but it won't work
+// until support for DMA ops' reconfiguration is added. The workaround therefore halves
+// the tile size for N as N/2.
+
+// Pack-peel-4-level tiling on 4x8 cores : the tile size remains maximum in this case.
+// PACK-PEEL-4-LEVEL{LITERAL}: #config = #iree_codegen.lowering_config<tile_sizes = [[32, 512, 0], [4, 4, 0], [0, 0, 1], [1, 1, 0]]>
```

Comment:
This seems you are still using 4 rows and 4 cols, so the above description that tiling on 4x8 cores is not accurate.